docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5-apache
+
+RUN a2enmod rewrite
+RUN apt-get update && \
+    apt-get install -y libxml2 libxml2-dev && \
+    docker-php-ext-install mysqli soap dom
+RUN yes | pecl install xdebug \
+    && echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" > /usr/local/etc/php/conf.d/xdebug.ini \
+    && echo "xdebug.remote_enable=on" >> /usr/local/etc/php/conf.d/xdebug.ini \
```

Comment:
Is this really a good idea? If you use this Dockerfile in production without modifications then XDebug will be on with remote enable. I would prefer if this was something you enabled via the `[XDEBUG_CONFIG` environment variable](https://xdebug.org/docs/remote#starting) or the like.